### PR TITLE
Home/landing/category page created

### PR DESCRIPTION
**Pull request by avindhawale**
Home page consists all the category of the products.
Please ignore images.

### CODE REVIEW

File: views/home/home.html [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../../assets/css/style.css" />
+    <link rel="stylesheet" href="../home/home.css">
+    <title>JSMonk ecommerce website - Home</title>
+    <script src="../../js/index.js"></script>
```
Comment (Devansu-Yadav) [inline views/home/home.html:11]: It is generally better to place JavaScript files just before the `body` tag closes if you are doing any DOM manipulations within your JS file. You can check this out - https://stackoverflow.com/questions/436411/where-should-i-put-script-tags-in-html-markup

**Review by Devansu-Yadav — APPROVED** [review]
Looks fine to merge. Just a small suggestion - You can use the 'preview PR before deploy' feature of Netlify to generate a preview of all the changes you have made in this PR. It helps reviewers to review your code easily.

File: views/home/home.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  background-color: var(--white);
+}
+.banner {
+  position: relative;
+  display: block;
+}
+.banner-desc {
+  position: absolute;
+  top: 100px;
+  right: 100px;
```
Comment (Hnh013) [inline views/home/home.css:22]: A suggestion: for future iterations, use variable for values that are use multiple times (like 2rem), also try to stick mostly to either px or rem , rest all looks good !

**Review by Hnh013 — APPROVED** [review]

File: views/home/home.html (hunk 1 above)
Comment (avindhawale) [reply views/home/home.html:11]: Sure, will do that.

File: views/home/home.css (hunk 2 above)
Comment (avindhawale) [reply views/home/home.css:22]: sure, will convert them into variables.

**Pull request merged**